### ISSUE TITLE
No images are displayed after selecting 'Kids' from the nav bar and there can be a hover effect to enhance the look of explore more 

### ISSUE THREAD

**Issue by thesri** [JiyaGupta-cs/ShopNex#2]
After selecting Kids from the nav bar nothing is dissplayed 
![Screenshot 2024-01-11 110424](https://github.com/JiyaGupta-cs/ShopNex/assets/98333833/2cf516cf-def9-4a17-b2e9-7a6942c3e670)
images can be added here 
Hover effect can be added to make the Enhance more look better 
![Screenshot 2024-01-11 110619](https://github.com/JiyaGupta-cs/ShopNex/assets/98333833/af90ccca-ffc1-4a42-9759-ea4d44319595)
Please assign me this issue under Jwoc 2024

**Comment by sanskritisahu21** [comment]
I'm eager to contribute to ShopNex , and after exploring the issues, I'm interested in working on this issue. Please assign this issue to me.

**Comment by thesri** [comment]
Ahhh I'm also a Mentee not a mentor so I myself am asking to assign this to me:)

**Comment by sanskritisahu21** [comment]
Oh so sorry i thought you are a mentor

**Comment by thesri** [comment]
> After selecting Kids from the nav bar nothing is dissplayed ![Screenshot 2024-01-11 110424](https://private-user-images.githubusercontent.com/98333833/295800368-2cf516cf-def9-4a17-b2e9-7a6942c3e670.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDUyNTg0NzcsIm5iZiI6MTcwNTI1ODE3NywicGF0aCI6Ii85ODMzMzgzMy8yOTU4MDAzNjgtMmNmNTE2Y2YtZGVmOS00YTE3LWIyZTktN2E2OTQyYzNlNjcwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE0VDE4NDkzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY3MDMzNGMxNWIwNDJjZTIzYzY4ZDk2YmU1MDJiM2E2Mjg4YjhjODdkOTY5ZDgwMzgzOTg5YzBlNjE4NTYxNTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.9Dhdh1BHwWbcTH-2MRYm3h2AVQbjgU8HPobkF5zHAl8) images can be added here Hover effect can be added to make the Enhance more look better ![Screenshot 2024-01-11 110619](https://private-user-images.githubusercontent.com/98333833/295800771-af90ccca-ffc1-4a42-9759-ea4d44319595.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDUyNTg0NzcsIm5iZiI6MTcwNTI1ODE3NywicGF0aCI6Ii85ODMzMzgzMy8yOTU4MDA3NzEtYWY5MGNjY2EtZmZjMS00YTQyLTk3NTktZWE0ZDQ0MzE5NTk1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE0VDE4NDkzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQyYWM3MWQ0YWI4YjljOTgwNDZlZjgyOTExZGQ2MDJhNDViODIwZjg3ODY3MjE2ZjhmNGRmMGNiZTY1YmNkZDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.kokDH-cBpFhDCdZsFoCnDQvaUpXGyAoLC11JKWDf2Ls) Please assign me this issue under Jwoc 2024

should i work on it? @sanskritisahu21

**Comment by pranjal12-web** [comment]
Hey @JiyaGupta-cs ...Please assign me this issue under JWoC'24

**Comment by JiyaGupta-cs** [comment]
@thesri Go ahead and add products for kids

**Comment by JiyaGupta-cs** [comment]
@thesri Are you working on it?

**Comment by thesri** [comment]
> @thesri Are you working on it?

Yes I will submit by the end of the day . I was little busy with my college stuff so sorry

**Comment by vishapraj** [comment]
Hi @JiyaGupta-cs please assign this issue to me.

**Comment by JiyaGupta-cs** [comment]
@sanskritisahu21 Assigning it to you

**Comment by sanskritisahu21** [comment]
where is kid section for adding the images and description.

**Comment by JiyaGupta-cs** [comment]
@sanskritisahu21 You need to create it just like men and women

**Comment by JiyaGupta-cs** [comment]
@sanskritisahu21 Are you working on it?

**Comment by JiyaGupta-cs** [comment]
@sanskritisahu21 Unassigning you due to inactivity

**Comment by VedikaKamane54** [comment]
Kindly assign me this issue

**Comment by JiyaGupta-cs** [comment]
@VedikaKamane54 Go ahead

**Comment by 20ankur07** [comment]
hey @JiyaGupta-cs  i completed the previous issue with button and was looking to try fixing this issue.  If you could assign it to me i could start working on it.
thanks
